src/formatDate.js

```diff
@@ -9,6 +9,34 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat.pop();
```

Comment:
It's not a good practice to mutate input arguments as it can lead to unexpected side effects. Instead of using `pop()` method which modifies the original array, consider using array indexing to get the last element of the array.